torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -1240,6 +1240,11 @@ void LoopNest::computeAt(Stmt* s, For* f) {
     Block* bb = dynamic_cast<Block*>(f->get_parent());
     bb->replace_stmt(f, new_f);
   }
+
+  // Mark the new temp buffer as requiring an alloc (it will be inserted as a
+  // part of prepareForCodegen.
+  temp_bufs_.push_back(temp_buf);
```

Comment:
nit: can we have this is as a single data structure? `std::vector<std::pair<..>>`